saleor/graphql/attribute/utils.py

```diff
@@ -671,6 +711,23 @@ def validate_standard_attributes_input(
     )
 
 
+def validate_date_time_input(
+    attribute: "Attribute",
+    attr_values: "AttrValuesInput",
+    attribute_errors: T_ERROR_DICT,
+    variant_validation: bool,
+):
+    is_date = attribute.input_type == AttributeInputType.DATE
+    is_date_time = attribute.input_type == AttributeInputType.DATE_TIME
+
+    if attribute.value_required and (
```

Comment:
Can we simplify this `if`(to reduce amount of `and` and `or`)?
This is only a suggestion but imho the current format is hard to read.
you can assign `is_date and not attr_values.date` to variable and pass it to if condition
you can also do something like this
```
if not attribute.value_required:
    return
```